pikaur/core.py

```diff
@@ -144,12 +144,12 @@ def isolate_root_cmd(cmd: List[str], cwd=None) -> List[str]:
     ]
     if cwd is not None:
         base_root_isolator += ['-p', 'WorkingDirectory=' + cwd]
-    if "http_proxy" in os.environ:
-        base_root_isolator += ['-E', 'http_proxy=' + os.environ.get('http_proxy')]
-    if "https_proxy" in os.environ:
-        base_root_isolator += ['-E', 'https_proxy=' + os.environ.get('https_proxy')]
-    if "ftp_proxy" in os.environ:
-        base_root_isolator += ['-E', 'ftp_proxy=' + os.environ.get('ftp_proxy')]
+    if "http_proxy" in os.environ and os.environ.get('http_proxy') is not None:
```

Comment:
just `if os.environ.get('http_proxy') is not None:` will be enough

Reply:
^ this comment is still actual: https://travis-ci.com/actionless/pikaur/builds/108801760#L1424